ModularTegustation/tegu_items/lc13/refinery/refinery.dm

```diff
@@ -0,0 +1,77 @@
+/obj/structure/refinery
```

Comment:
Wouldn't making this a machine allow you to utilize process instead of repeatedly using a timer?

Reply:
Jesus Christ what? Actually?

Reply:
Yep, I keep an eye on our MC sometimes,  Machinery and Mobs are our two biggest issues.